DataDefinitions/Vehicle.cs

```diff
@@ -49,7 +49,7 @@ public static Vehicle FromJson(int subslot, dynamic json)
         {
             if (json is null) { return null; }
 
-            string edName = (string)json["EDName"];
+            string edName = (string)json["name"];
```

Comment:
Are you 100% sure about this and the related changes to the test data? I was definitely seeing 'EDName' in the json.

Reply:
Yes, this is correct. Part of json from my Python for laucnhbays...
```    Slot09_Size4: {
        SubSlot0: {
            name: "testbuggy",
            locName: "SRV Scarab",
            rebuilds: 1,
            loadout: "starter",
            loadoutName: "Starter"
        },
        SubSlot1: {
            name: "testbuggy",
            locName: "SRV Scarab",
            rebuilds: 1,
            loadout: "starter",
            loadoutName: "Starter"
        }
    }
}
```

Reply:
I guess I must have been looking at old JSON then. Le sigh.

Reply:
I can confirm @Hoodathunk's json format.